### PR TITLE
feat(tray): add settings for hiding current task at Mac menu bar icon

### LINKED ISSUE #495: [Suggestion] Add toggle to display/hide task name in macOS menu bar
### Your Environment
* Version used: 5.4.3 (5.4.3.1638) via homebrew
* Operating System and version: macOS 10.14.6

The app may provide a toggle option as part of its settings to show/hide task name in the menu bar.

Currently, the app takes a lot of screen estate in the menu bar. This makes other useful apps in the menu bar unreachable. Its not much of an issue on large monitors but it is on a laptop screen.

superProductivity_issue

_App in menu bar hides other icons making them unusable._

### PR DESCRIPTION
**Pull request by fuerst**
# Description

Adding a new setting at General > Misc, called _Show current task in the tray / status menu (desktop only)_.
If setting is true the indicator string at electron/indicator.ts will not be created.

BTW: I used BabelEdit to edit the language files. This automatically consolidated/fixed
all language files to contain the same keys. Also fixed some missing comma.

## Issues Resolved

#495

## Check List

- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by johannesjo** [comment]
Thank you very much! I will test this asap.

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -83,12 +83,40 @@
       }
     },
     "DROPBOX": {
+      "C": {
```
Comment (johannesjo) [inline src/assets/i18n/en.json:86]: I wonder where these are all coming from. Could you please remove all those empty translations?

**Review by johannesjo — CHANGES_REQUESTED** [review]
Maybe you can remove all these empty translation keys. Normally I only adjust the en.json and translate the rest automatically. So if you have any new translations, it's important to also add them to the en.json. The rest of the files can be ignored unless you want to manually add the translations there too.

File: src/assets/i18n/en.json (hunk 1 above)
Comment (fuerst) [reply src/assets/i18n/en.json:86]: BabelEdit was adding them, usually I trust it to do the right thing. I remove it it though and just add the new key.

**Comment by johannesjo** [comment]
Thanks for the adjustments. Normally Babeltranslate should work fine, and I appreciate the extra effort, but this did get messed up somehow. I'll investigate this after the merge.

**Comment by johannesjo** [comment]
Thank you very much!

**Comment by fuerst** [comment]
Thanks for merging. Glad to help!

**Pull request merged**